components/StyledCheckbox.js

```diff
@@ -63,7 +63,6 @@ const CheckboxContainer = styled.div`
     cursor: pointer;
     margin: 0;
     margin-left: 2.2rem;
-    z-index: 9;
```

Comment:
Why remove this? Did you test to make sure there was no side-effect?

Reply:
Added this back, a later fix that was successfully targeting the autocomplete dropdown to be on top fixed it. Wrong approach to mess with this here. Thanks for noticing!